website/components/LandingPage/index.tsx

```diff
@@ -253,16 +254,11 @@ export default function LandingPage(): ReactElement {
   const onClickLoad = (appProps: AppDataProps): void => {
     // TODO: Make URL search params from the appProps and append it to the viewer URL so the URL can be shared directly.
     // Alternatively, AppWrapper should manage syncing URL and viewer props.
-    const url = appProps.imageUrl;
-    if (Array.isArray(url)) {
-      navigation(`/viewer?url=${encodeURIComponent(url.join(","))}`, {
-        state: appProps,
-      });
-    } else {
-      navigation(`/viewer?url=${encodeURIComponent(url)}`, {
-        state: appProps,
-      });
-    }
+    const urls = (appProps.imageUrl as MultisceneUrls).scenes ?? [appProps.imageUrl];
+    const sceneUrlsUnencoded = urls.map((scene) => (Array.isArray(scene) ? scene.join(",") : scene));
+    navigation(`/viewer?url=${encodeURIComponent(sceneUrlsUnencoded.join(" "))}`, {
```

Comment:
If this code is copied with `AppWrapper` maybe it's worth putting in a function?